README.md

```diff
@@ -1,7 +1,6 @@
 # Tesla BLE MQTT Docker
 
 Send commands via MQTT to a Tesla car using Bluetooth Low Energy (BLE).
```

Comment:
We need to put your coffee somewhere :)

Reply:
I think we drop it, no one ever clicks it, and it's our coffee now :)